client/me/purchases/concierge-banner/index.jsx

```diff
@@ -42,86 +42,91 @@ class ConciergeBanner extends Component {
 	getBannerContent() {
 		const { bannerType, translate } = this.props;
 
-		let headerText;
-		let mainText;
+		let title;
+		let description;
 		let buttonText;
-		let buttonHref;
-		let illustrationUrl;
+		let event;
 
 		switch ( bannerType ) {
 			case CONCIERGE_HAS_UPCOMING_APPOINTMENT:
-				headerText = translate( 'Your appointment is coming up!' );
-				mainText = translate(
-					'Get ready with your questions for your upcoming Quick Start session appointment.',
+				title = translate( 'Your session is coming up soon' );
+				description = translate(
+					'Your {{supportLink}}Quick Start support session{{/supportLink}} is approaching. Get ready for your one-to-one with our Happiness Engineer.',
```

Comment:
Another opportunity for later iteration: we can show the date and time for the next scheduled session, and possibly the site the session is for.